web/scss/ui/_reset.scss

```
@@ -67,6 +67,11 @@ html {
         color: var(--inverse);
         background-color: var(--ac-4);
     }
+
+    blockquote {
+        padding-left: 0.5em;
+        border-left: 0.5em solid var(--bg-4);
```

I think it's best to avoid using `em` units for anything other than `font-size`.
```suggestion
        padding-left: 0.5rem;
        margin-bottom: 0.5rem;
        border-left: 0.25rem solid var(--bg-4);
```